android/KMEA/build.sh

```diff
@@ -136,7 +129,7 @@ if [ "$DO_BUILD" = true ]; then
     ./build.sh $KMWFLAGS
 
     if [ $? -ne 0 ]; then
-        die "ERROR: keymanweb build failed. Exiting"
+        builder_die "ERROR: keymanweb build failed. Exiting"
```

Comment:
I'll point out that `builder_die` is solely used by the older build scripts in the repo, not by any of the ones using the new common builder functions.  Unless I'm mistaken?

Reply:
There is a role for `die` and `builder_die` where other conditions are not met. But not typically for testing exit values. But see [my comment](https://github.com/keymanapp/keyman/pull/7365#issuecomment-1260267926) about changes to existing scripts.